integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/CloudbreakClusterTestConfiguration.java

```diff
@@ -24,7 +24,7 @@
     @BeforeSuite
     public void getAllClusterDefinitions() throws Exception {
         given(CloudbreakClient.created());
-        when(ClusterDefinition.getAll());
+//        when(ClusterDefinitionTestAction.getAll());
```

Comment:
If this config class is no longer needed, then delete it completely. There is already too much garbage in the int tests.

Reply:
After all of the old tests are rewritten we could delete dead codes